### ISSUE TITLE
GQN results in your repo's README

### ISSUE THREAD

**Issue by Amir-Arsalan** [iShohei220/torch-gqn#9]
I recently used [another](https://github.com/wohlert/generative-query-network-pytorch) implementation of GQNs but could [not reproduce](https://github.com/wohlert/generative-query-network-pytorch/issues/23) [or get something close to] the results that the GQN paper shows. In fact, the loss plateaus after a couple of epochs and doesn't go down. I'm thinking that the code I used might have some bugs that are hard to identify and fix. So I'm thinking of using your implementation but I thought it might be good to check with you on whether the reconstructions you are showing through your repo's README are obtain from a model trained using your code or not?

**Comment by Amir-Arsalan** [comment]
I forgot to confirm that your implementation works :)
